src/ophys_etl/pipelines/segment_binarize_pipeline.py

```diff
@@ -1,54 +1,96 @@
+import json
+from pathlib import Path
+import tempfile
+
+import marshmallow
 import argschema
-from ophys_etl.transforms.suite2p_wrapper import Suite2PWrapper
-from ophys_etl.transforms.convert_rois import BinarizerAndROICreator
-from ophys_etl.schemas.fields import H5InputFile
 
-import tempfile
-import pathlib
-import json
+
+from ophys_etl.transforms.convert_rois import (
+    BinarizeAndCreateROIsInputSchema, BinarizerAndROICreator)
+from ophys_etl.transforms.suite2p_wrapper import (Suite2PWrapper,
+                                                  Suite2PWrapperSchema)
+
+
+class ConvertROIsInputSchema(BinarizeAndCreateROIsInputSchema):
+
+    # Override the default "suite2p_stat_field" required parameter of the
```

Comment:
nice solution!

Reply:
It was actually @kschelonka 's idea. I had unfortunately developed a bit of tunnel vision regarding ways to solve the problem.